interface/forms/care_plan/new.php

```diff
@@ -26,8 +26,8 @@
 $returnurl = 'encounter_top.php';
 $formid = (int)($_GET['id'] ?? 0);
 if (empty($formid)) {
-    $sql = "SELECT id, encounter FROM `form_care_plan` WHERE pid = ? AND encounter = ?  LIMIT 1";
```

Comment:
@sjpadgett  Isn't the id and the forms.form_id the same?  I can see where you are checking to only grab ones that are not deleted, but just curious here.

Reply:
they are the same it's just we don't show deletes in the form_care_plan or form_clinical_notes when deleting from encounter, only the master forms table.